src/operators/dyn_op.ml

```diff
@@ -138,11 +139,11 @@ let () =
           "Whether the source is infallible or not (be careful when setting \
            this, it will not be checked by the typing system)." );
       ( "resurection_time",
-        Lang.float_t,
-        Some (Lang.float 1.),
```

Comment:
Do you really want to change the default value here?

Reply:
Good catch thks